### PR TITLE
[DEV-3309] Add hl7 type and trigger to received log

### PR DESCRIPTION
**Pull request by taylorrietz**
```json
{
    "asctime": "2023-10-10 18:19:54,531",
    "levelname": "INFO",
    "name": "hl7_listener.main",
    "filename": "main.py",
    "lineno": 53,
    "dd.service": null, "dd.env": null, "dd.version": null, "dd.trace_id": null, "dd.span_id": null,
    "message": "HL7LLOG003: HL7 Listener received a message of type: ADT^A01"
}
```

### CODE REVIEW

File: src/main/py/hl7_listener/main.py [hunk 1]
```diff
@@ -44,10 +44,13 @@ async def process_received_hl7_messages(hl7_reader, hl7_writer):
         hl7_message = None
         while not hl7_reader.at_eof():
             hl7_message = await hl7_reader.readmessage()
-            logger.info(logging_codes.HL7_MLLP_MSG_RECEIVED)
             # This may not be needed since the hl7_mllp sender should fail if the message
             # was not valid hl7 message.
-            hl7.parse(str(hl7_message))
+            _parsed = hl7.parse(str(hl7_message))
```
Comment (dixonwhitmire) [inline src/main/py/hl7_listener/main.py:49]: We could move the `_parsed` and other `_` variables into the `if`. That appears to be the only place that they are used.

**Review by dixonwhitmire — COMMENTED** [review]
Hey @taylorrietz - I just had a suggestion regarding where we're setting the "_" variables.
Thanks!

**Review by sajal-g — DISMISSED** [review]

**Review by keelamp — COMMENTED** [review]
Not required, but if it's not too much of a hassle, can you add this to the Justfile: 
```
test-coverage *args:
    poetry run pytest {{args}} --cov-report term-missing --cov=src/qcc_dicom_tag_remover --cov-fail-under=80 tests/
```

If this is for a hotfix, then don't worry and remove the 80% threshold requirement from the jira.

**Comment by taylorrietz** [comment]
> Not required, but if it's not too much of a hassle, can you add this to the Justfile:
> 
> ```
> test-coverage *args:
>     poetry run pytest {{args}} --cov-report term-missing --cov=src/qcc_dicom_tag_remover --cov-fail-under=80 tests/
> ```
> 
> If this is for a hotfix, then don't worry and remove the 80% threshold requirement from the jira.

Sure! I have this locally, just forgot to push 🤦

File: src/main/py/hl7_listener/main.py (hunk 1 above)
Comment (taylorrietz) [reply src/main/py/hl7_listener/main.py:49]: We're actually logging them either way, whether or not the dd span has been injected via the decorator to the function's `ddspan` param.

**Review by keelamp — APPROVED** [review]
COVERA(GE) HEALTH

**Pull request merged**